src/qt/rpcconsole.cpp

```diff
@@ -9,13 +9,14 @@
 #include <qt/rpcconsole.h>
 #include <qt/forms/ui_debugwindow.h>
 
+#include <chainparams.h>
+#include <interfaces/node.h>
```

Comment:
On My Machine™, it seems only `#include <interfaces/node.h>` is needed?

Reply:
That commit only adds `#include <qt/peertablesortproxy.h>`, and other `include`s are sorted only. I'd leave the need of the `interfaces/node.h` header out of this pr scope.

Reply:
Sorry, missed the fact that it's a sort.